### DISCUSSION TITLE
Suggested pipx installation method in docs fails for adapters

### DISCUSSION THREAD

**Discussion by jonbiemond** [tconbeer/harlequin#483] [Q&A, Troubleshooting, Support] [answered]
**Before Proceeding, please acknowledge**:
- [x] I have consulted the [Troubleshooting guide](https://harlequin.sh/docs/troubleshooting/index).
- [x] I have searched Issues and Discussions in this repo.
- [x] Feature requests should be initiated as Discussions. This is a bug report.

> [!NOTE]
> Bug report regarding the docs at [https://harlequin.sh/docs/postgres/index](https://harlequin.sh/docs/postgres/index), not Harlequin's behaviour.
> Figured discussions might be appropriate since it won't be addressed by a PR in this repo.

**Describe the bug**
The default `pipx` instructions in the docs for installing Harlequin with an adapter fails, because no apps are associated with the adapter specific package.

**To Reproduce**
Install adapter package with pipx:
```bash
pipx install harlequin-postgres
```

**Expected behavior**
Instead have docs suggest installing Harlequin as an extra for pipx:
```bash
pipx install harlequin[postgres]
```

**Actual behavior**
`pipx` fails to properly install Harlequin because no app is found, it treats `harlequin-postgres` as a library.
```
PS C:\Users\jonathan.biemond> pipx install harlequin-postgres
Note: Dependent package 'harlequin' contains 1 apps
  - harlequin.exe
Note: Dependent package 'rich-click' contains 1 apps
  - rich-click.exe
Note: Dependent package 'markdown-it-py' contains 1 apps
  - markdown-it.exe
Note: Dependent package 'pygments' contains 1 apps
  - pygmentize.exe
Note: Dependent package 'shandy-sqlfmt' contains 2 apps
  - sqlfmt.exe
  - sqlfmt_primer.exe
Note: Dependent package 'tqdm' contains 1 apps
  - tqdm.exe
Note: Dependent package 'numpy' contains 1 apps
  - f2py.exe

No apps associated with package harlequin-postgres. Try again with '--include-deps' to include apps of dependent packages, which are listed above. If you are attempting to install a library, pipx should not
be used. Consider using pip or a similar tool instead.
```

**Contributing**
Are you interested in contributing a fix?
- [x] Yes
- [ ] Maybe
- [ ] No

**Additional context**
```
PS C:\Users\jonathan.biemond> pipx --version
1.4.3
```

Installing with `pipx` as an extra or via `inject` works fine. I haven't tested this for anything but the `postgres` adapter but I assume the same holds true.

Can you tell us more about your system?
- Shell: PowerShell
- Terminal: Windows Terminal
- OS of the shell: Windows 10
- Default locale / language for your OS/Terminal/Shell: en_US/UTF-8

How did you install Harlequin?
- [x] pipx
- [ ] conda/mamba
- [ ] pip (global install)
- [ ] pip + venv/virtualenv or pipenv
- [ ] Poetry
- [ ] uv/Rye
- [ ] other: _____________

**Comment by tconbeer** [comment, marked as answer]
Thanks, agree with the issue and the solution

**Reply by jonbiemond** [reply]
👍 Awesome, is there a way to contribute to the documentation? I can't seem to find it on the site or in the repo.

**Reply by tconbeer** [reply]
the docs site is this repo:
https://github.com/tconbeer/harlequin-web

Would also love PRs against the READMEs in each adapter's repos, e.g., https://github.com/tconbeer/harlequin-postgres, https://github.com/tconbeer/harlequin-mysql, https://github.com/tconbeer/harlequin-odbc
